doc/configuration.rst

```diff
@@ -580,6 +580,31 @@ Arguments:
   - boot_expression (str): optional, regex to match the uboot start string
     defaults to "U-Boot 20\d+"
 
+SmallUBootDriver
+~~~~~~~~~~~
+A SmallUBootDriver interfaces with stripped-down UBoot variants that are
+sometimes used in cheap consumer electronics. 
```

Comment:
Please describe when you would use this compared to the normal UBootDriver. Also update the UBootDriver docs to be explicit about which commands it needs to work.